examples/cli/scripts/multiple_invocations.py

```diff
@@ -32,7 +33,7 @@ def forward(self, x):
 
 
 # Instantiate model and generate inputs
-input_features = 11
+input_features = 12
```

Comment:
This was a bugfix; the models were identical across two of the example scripts, which made `turnkey scripts/*.py` malfunction.